lib/api_calls/faucet_api.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:convert';
+
+import '../global.dart';
+
+class FaucetApi {
+  getFreeTokens(String address) async {
```

Comment:
we should add return types to all methods, here `void`. I'll also make sure to add some linter later on

Reply:
Updated all functions to have return types 🙌